requirements-dev.txt

```diff
@@ -7,6 +7,7 @@ pytest-cov~=2.11.1
 pytest-recording~=0.11.0
 pytest-console-scripts~=1.1.0
 recommonmark~=0.7.1
+requests-mock~=1.9.3
```

Comment:
NBD but this will conflict with #118 (maybe auto-deconflictable?).

Reply:
Yeah, I'm not sure if git will just resolve it automatically since it's the same change in the same place, but either way it should be easy to resolve. I can rebase one of these PRs once the other has been merged if it helps.